### ISSUE TITLE
User preferences language is different from opencms-system.xml

### ISSUE THREAD

**Issue by arrowwind** [alkacon/opencms-core#675]
For 11.0.1 the Locale can be defined in the opencms-system.xml, But the User preferences language
is different. It still use the old way.

So it can be fix by changing the code in src/org/opencms/workplace/CmsWorkplaceManager.java

change ` private List<Locale> initWorkplaceLocales(CmsObject cms) {    **** }`

to `
private List<Locale> initWorkplaceLocales(CmsObject cms) {  
 ArrayList<Locale> result = new ArrayList<>(OpenCms.getLocaleManager().getAvailableLocales());
        result.sort(CmsLocaleComparator.getComparator());
        return result;
}
`

**Comment by tHerrmann** [comment]
The locales defined in the opencms-system.xml are not related to the locales available as workplace locales in the user preferences. The first are used for the contents of the websites you manage with OpenCms the later is the user interface language for your editor users.
In case you want to restrict the available workplace locales, you can remove the JARs org.opencms.locale.[locale] from the WEB-INF/lib folder in the real file system and restart your servlet-container. The english locale can not be removed, it is always available.

**Comment by arrowwind** [comment]
Yes, you are right. I agree that the contents locale is different from the editor interface. 

Now there is english and other 6 locales right now which the end user can choose in the user perferences. When the admin user adds a new user or edits the exist user, the languages the admin user can choose the ones which exist in the opencms-system.xml. For the end user view, that is confusing. I can choose 6 language in my user preferences but only 2 in the User Management.  And if all of them are update to date that will be fine, for en and de they are fully translated for others I don't think so.

As the website developer,  to deliver the untranslated content is not a professional way. So before deliver the whole thing to the end user, he will delete the unused locales if he knows to delete the jar in the lib folder or to debug the whole program to find out how to do it. Why not make the two settings the same. So the new opencms developer will not consider this thing.

Thanks for reading.
